### PR TITLE
added terminal commands under backend topic

### PR DESCRIPTION
**Pull request by ashutoshsao**
the video teaches us how to use terminal and make the most out of the powerful bash script

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,7 +47,8 @@ To add resources or to contribute read the [Contribution-guide](https://github.c
 ## Index
 
 - [Android Development](#android-development)
-- [Backend](#backend)  
+- [Backend](#backend)
+  * [Shell Commands](#shell commands)
```
Comment (Ashish-khanagwal) [inline README.md:51]: Hey, Please replace this with `[Shell Commands](#shell-commands)` so, that it can redirect to its required location

**Review by Ashish-khanagwal — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ashutoshsao) [reply README.md]: please review the changes i made

**Comment by ashutoshsao** [comment]
but still it isn't redirecting to the location, i am unable to understand why it's happening.

**Comment by Ashish-khanagwal** [comment]
Don't worry, I resolved it on my end

Thank you for contributing

**Review by Ashish-khanagwal — APPROVED** [review]

**Pull request merged**